src/pages/gen2/start/quickstart/vite-react-app/index.mdx

```diff
@@ -0,0 +1,270 @@
+    const { data } = await client.models.Todo.list();
+    setTodos(data);
+  }
+
+  useEffect(() => {
+    listTodos();
+  }, []);
+
+  return (
+    <div>
+      <h1>Todos</h1>
+
+      <ul>
+        {todos.map((todo) => (
+          <li key={todo.id}>{todo.content}</li>
+        ))}
+      </ul>
+    </div>
+  );
+}
+```
+
+In `src/App.tsx`, import `components/TodoList` and render it:
+
+```ts title="src/App.tsx"
+import { withAuthenticator } from "@aws-amplify/ui-react";
+import "@aws-amplify/ui-react/styles.css";
+import TodoList from "./components/TodoList";
+
+function App() {
+  return (
+    <>
+      <h1>Hello, Amplify 👋</h1>
+      <TodoList />
+    </>
+  );
+}
+
+export default withAuthenticator(App);
+```
+
+Once you save the file and navigate back to `http://localhost:5173`, you should see a blank page for now because you have only an empty list of to-dos.
+
+### Create a new to-do item
+
+Let's update the return statement of the `components/TodoList` component to have a button for creating a new to-do list item, prompting the user to add the title and description. In a production app, you would want to create a full form, and run the create method on form submission.
+
+```tsx title="components/TodoList.tsx"
+// ...
+  return (
+    <div>
+      <h1>Todos</h1>
+      <button onClick={async () => {
+        // create a new Todo with the following attributes
+        const { errors, data: newTodo } = await client.models.Todo.create({
+          // prompt the user to enter the title
+          content: window.prompt("title"),
+          done: false,
+          priority: 'medium'
+        })
+        console.log(errors, newTodo);
+      }}>Create</button>
+
+      <ul>
+        {todos.map((todo) => (
+          <li key={todo.id}>{todo.content}</li>
+        ))}
+      </ul>
+    </div>
+  );
+}
+```
+
+Create a couple of to-dos, then refresh the page to see them. You can also subscribe to new to-dos in your `useEffect` to have them live reload on the page.
+
+```js
+useEffect(() => {
+  const sub = client.models.Todo.observeQuery().subscribe(({ items }) =>
+   setTodos([...items])
+  );
+
+  return () => sub.unsubscribe();
+}, []);
+```
+
+### Terminate dev server
+
+Go to `localhost` in the browser to make sure you can now log in and create and list to-dos. You can end your development session by shutting down the frontend dev server and cloud sandbox. The sandbox prompts you to delete your backend resources. While you can retain your backend, we recommend deleting all resources so you can start clean again next time.
+
+## Deploy and host a fullstack branch
+
+Now that your app is working, let's deploy it to a shared fullstack branch so you can share the project with your team. Amplify offers a fully managed hosting service with CI/CD built in, making it easy to set up production and staging environments with Git branches. In Gen 2, every Git branch in your repository maps 1:1 to a fullstack branch in Amplify.
```

Comment:
similar styleguide nit as before for "you" and "let's", but other guides are the same. no suggestion https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md?rgh-link-date=2024-02-01T18%3A55%3A26Z#5-we-are-the-authors-you-is-the-reader

Reply:
adjusted wording

https://github.com/aws-amplify/docs/pull/6870/commits/9f39934c65cc5acc2286c13452e6fe423b1630a8